### ISSUE TITLE
Google Pay Tap no longer works 

### ISSUE THREAD

**Issue by kayslay177** [kdrag0n/safetynet-fix#265]
**Describe the bug**
When I go to add a card to Google Wallet it says "your phone does not security standards to make contactless payments. It may be rooted or running uncertified software."

**To reproduce**
Go to Google Wallet, click add card, fill out care information, click continue 

**Expected behavior**
Until a week ago, adding a card to Google Wallet was possible, allowing for contactless payment with my phone 

**Device info**
Pixel 6
Android 13

**Comment by minilandl** [comment]
Same issue here 
![Screenshot_20230220-154318_Yet_Another_SafetyNet_Attestation_Checker](https://user-images.githubusercontent.com/47686364/220062339-b74fd702-ae1d-468b-afb7-138ace3c29d8.png)

Tried using the latest universal safeteynet fix  and reflashed latest stable magisk still CTS pass false .
 Also tried using latest magisk hide props config with devices simulation to a Pixel 6

Happened last night so my device must have received the safeteynet update . 

I am using a mi10t with  Evolution X Android 13

**Comment by tzkoshi** [comment]
same here - AOSP Android 13, Xiaomi mi 11t pro is the phone.
CTS pass fails.

**Comment by fastleo63** [comment]
Me, too.
POCO F3, MIUI 13.0.9 Global. 
Google changed something with their Play Services new update (yesterday was working flawlessly). 

![Screenshot_2023-02-20-12-05-55-785-edit_rikka safetynetchecker](https://user-images.githubusercontent.com/38861089/220091057-2efc489a-351e-47ea-a3a3-d65fb4af34e1.jpg)

**Comment by Displax** [comment]
https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

**Comment by minilandl** [comment]
I was able to get it working again by using this fork of universal safeteynet fix seems to fix attestation https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

After installing I can pass safetynet and Google wallet works like before 
![Uploading Screenshot_20230220-193730_Yet_Another_SafetyNet_Attestation_Checker.png…]()

**Comment by kayslay177** [comment]
> I was able to get it working again by using this fork of universal safeteynet fix seems to fix attestation https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2
> 
> After installing I can pass safetynet and Google wallet works like before ![Screenshot_20230220-193730_Yet_Another_SafetyNet_Attestation_Checker](https://user-images.githubusercontent.com/47686364/220106169-2dc24dab-27f6-43de-adcc-b25be3904699.png)

Damn it, still not working for me :( Did you just install it and were good to go, or did you have to do anythign else?

**Comment by kayslay177** [comment]
> https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2

Didn't fix for me :( I just installed and rebooted, do I need to do anything else?

**Comment by Displax** [comment]
> > https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2
> 
> Didn't fix for me :( I just installed and rebooted, do I need to do anything else?

Did you pass integrity test?

**Comment by kayslay177** [comment]
> > > https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2
> > 
> > 
> > Didn't fix for me :( I just installed and rebooted, do I need to do anything else?
> 
> Did you pass integrity test?

Oh just figured out how to check.....Basic integrity PASS, CTS profile match PASS

**Comment by Displax** [comment]
> > > > https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2
> > > 
> > > 
> > > Didn't fix for me :( I just installed and rebooted, do I need to do anything else?
> > 
> > 
> > Did you pass integrity test?
> 
> Oh just figured out how to check.....Basic integrity PASS, CTS profile match PASS

https://play.google.com/store/apps/details?id=gr.nikolasspyr.integritycheck

**Comment by kayslay177** [comment]
> > > > > https://github.com/Displax/safetynet-fix/releases/tag/v2.4.0-MOD_1.2
> > > > 
> > > > 
> > > > Didn't fix for me :( I just installed and rebooted, do I need to do anything else?
> > > 
> > > 
> > > Did you pass integrity test?
> > 
> > 
> > Oh just figured out how to check.....Basic integrity PASS, CTS profile match PASS
> 
> https://play.google.com/store/apps/details?id=gr.nikolasspyr.integritycheck

It doesn't pass strong integrity

It also seems the message on my Wallet app has changed. It no longer says I can't use it cause my decide doesn't allow it, it now says "couldn't finish setting up tap to pay. Something went wrong when setting up your card. Try again or add another card"

**Comment by Displax** [comment]
> It doesn't pass strong integrity

That's how it should be. So just clear data of Google Play Services.

**Comment by kayslay177** [comment]
> > It doesn't pass strong integrity
> 
> That's how it should be. So just clear data of Google Play Services.

This is what I get when adding card to wallet now 
![Screenshot_20230220-105911](https://user-images.githubusercontent.com/37699809/220182135-2ee26823-5701-4cc5-b8c0-61e4726db86b.png)

**Comment by Displax** [comment]
> This is what I get when adding card to wallet now

You need to clearing GMS data if all test passed.

**Comment by kayslay177** [comment]
> > This is what I get when adding card to wallet now
> 
> You need to clearing GMS data if all test passed.

How do I do that?

**Comment by kayslay177** [comment]
> > How do I do that?
> 
> Settings -> Apps -> All Apps -> System processes -> Google Play services -> Data and cache -> Manage Data -> Dell all data. Reboot.

Cool, got one card to work so it's obviously fixed it, prob just need to wait on the others cause this has hapepned before and it just involved wiatingfor bank to cleark token or some shit. Thanks so much for your help

**Comment by kayslay177** [comment]
> > How do I do that?
> 
> Settings -> Apps -> All Apps -> System processes -> Google Play services -> Data and cache -> Manage Data -> Dell all data. Reboot.

alright, all goodto go, got my 3 cards added. Thanks a ton!

**Comment by TheBestPessimist** [comment]
Hey @Displax , Is your mod supposed to also hide zygisk from banking apps?

My Banking App: ING Homebank Romania: https://play.google.com/store/apps/details?id=ro.ing.mobile.banking.android.activity&hl=ro&pli=1


Right now i'm in this catch 22 situation where
- enable zygisk + your mod
	- i can use google wallet
	- i cannot use ING because it detects zygisk
- disable zygisk +  [this](https://github.com/HuskyDG/MagiskHide) mod
	- i cannot use google wallet
	- i can use ING bank

So no matter what i try, i cannot add my card into wallet as it doesn't work due to one of the apps detecting root.
